### PR TITLE
refactor(telephony.alias): remove old custom class of oui-header-tabs

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-1997

### CODE REVIEW

File: client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
         <!-- LINE ACTIONS -->
         <nav role="navigation">
-            <oui-header-tabs class="oui-header-tabs">
+            <oui-header-tabs>
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: Is it still used ? If not, you can remove it ;)

Comment (JeremyDec) [reply client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: Yes it's still used

Comment (antleblanc) [reply client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: The [`<oui-header-tabs>`](https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-header-tabs/src/header-tabs.html#L1) template has already the `<nav>` element. You can get rid of.

According to your commit message, this is not type `style` but sounds more like a `refactor` or `fix`. Isn't it?

Comment (JeremyDec) [reply client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: Ok, my bad, I thought @FredericEspiau was saying if `oui-header-tabs` element was still used

Comment (JeremyDec) [reply client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: @antleblanc => We should change https://github.com/ovh-ux/ovh-ux-guidelines/blob/master/.github/CONTRIBUTING.md#tldr to be clear

Comment (antleblanc) [reply client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html]: In my point of view, this repository isn't fully up-to-date. It has been published as a draft.

We probably have multiple choice such as:
- Maintain an up-to-date conventions/guidelines.
- Stay aligned with the actual framework documentation which offer best practices.

For now `ovh-ux/ovh-ux-guidelines` isn't correctly documented.

**Pull request merged**